src/treemodel/nodelist.js

```diff
@@ -38,16 +83,16 @@ export default class NodeList {
 	 *
 	 *		let nodeList = new NodeList( new Text( 'foo', [ new Attribute( 'bar', 'bom' ) ] ) );
 	 *		nodeList.length; // 3
-	 *		nodeList.get( 0 ).getAttr( 'bar' ); // 'bom'
-	 *		nodeList.get( 1 ).getAttr( 'bar' ); // 'bom'
-	 *		nodeList.get( 2 ).getAttr( 'bar' ); // 'bom'
+	 *		nodeList.get( 0 ).attrs.get( 'bar' ); // 'bom'
+	 *		nodeList.get( 1 ).attrs.get( 'bar' ); // 'bom'
+	 *		nodeList.get( 2 ).attrs.get( 'bar' ); // 'bom'
 	 *
 	 *		let nodeListA = new NodeList( 'foo' );
 	 *		let nodeListB = new NodeList( nodeListA );
 	 *		nodeListA === nodeListB // true
 	 *		nodeListB.length // 3
 	 *
-	 * @param {treeModel.Node|treeModel.Text|treeModel.NodeList|String|Iterable} nodes List of nodes.
+	 * @param {treeModel.Node|treeModel.Text|String|treeModel.NodeList|Iterable} nodes List of nodes.
```

Comment:
Since we use this list of parameters many times, it may be useful to define it as a type using @typedef, the same way we decided to handle enums?